common/lib/misc.c

```diff
@@ -21,6 +21,18 @@ UINT32 efi_desc_ver = 0;
 
 bool editor_enabled = true;
 
+const char *get_arch_name(void) {
+#ifdef __i386__
+    return "i386";
+#elif defined (__x86_64__)
+    return "x86_64";
+#elif defined (__aarch64__)
+    return "aarch64";
+#else
+    return "unknown";
```

Comment:
I feel like this should be an `#error` just to force anyone porting Limine to a new architecture to handle it